static/lang/en.json

```diff
@@ -3928,6 +3932,8 @@
         "TraitDescriptionWitch": "This indicates abilities from the witch class.",
         "TraitDescriptionWizard": "This indicates abilities from the wizard class.",
         "TraitDescriptionWood": "Effects with the wood trait conjure or manipulate wood. Those that manipulate wood have no effect in an area without wood. Creatures with this trait consist primarily of wood or have a connection to magical wood.",
+        "TraitBottledBreath": "Bottled Breath",
```

Comment:
Alphabetize

Reply:
It's sorted with like. Should I sort the whole list?

Reply:
The whole list should be alphabetized, I did a sort on it not long ago but for the love of everything holy don't sort the whole thing right now 😄 My RoE branch has to be able to merge without me throwing a monitor out a window on the fifteenth merge conflict.